### ISSUE TITLE
role gets lost over time. reverts to default?

### ISSUE THREAD

**Issue by cfurst** [cfurst/fintojs#13]
### steps to reproduce
1. start fintojs
1. set a role
1. use role and then let idle for about 30mins or hour
1. try to use the same role.

you will get access denied if you are trying to access something that is not allowed and if you try to do something you are allowed, you are clearly not getting data from that role's namespace.

### Example:
This occured after having nhl set as the active role yesterday and then coming back to today we have:
```bash
BAM-035034-CF:~ furst$ finto-set nhl
{
    "active_role": "nhl"
}

BAM-035034-CF:~ furst$ aws lambda list-functions

An error occurred (AccessDeniedException) when calling the ListFunctions operation: User: [sic] is not authorized to perform: lambda:ListFunctions on resource: *

BAM-035034-CF:~ furst$ aws s3 ls
... partially removed ...
2016-08-18 10:59:36 trail-testing-bucket
```
role is now stuck. You can't change it to another role and any request that is not allowed your default user is blocked.
*Workaround:* The only workaround at this point is to restart the server.

**Comment by cfurst** [comment]
It would seem that the expiration date returned by STS is not the expiration of the assume role session. So it is not a good guide to go by when to renew the assume role session. There is a way to get the MaxDurationInSeconds parameter of any role which we could use as a guide for when to renew the session, but that would mean an extra call to retrieve the Role meta data and use that otherwise just set an hour as this is the default minimum?

**Comment by cfurst** [comment]
I'm thinking assumeRole is not the right way to do this.. there is a better way to get TemporaryCredendials.. see where I'm going with this? 

https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/TemporaryCredentials.html

**Comment by cfurst** [comment]
Fixed in 1.1.0 
https://github.com/cfurst/fintojs/releases/tag/1.1.0
